src/imitation/algorithms/preference_comparisons.py

```diff
@@ -1283,9 +1363,8 @@ def train(
             # at the end of training (where the reward model is presumably best)
             if i == self.num_iterations - 1:
                 num_steps += extra_timesteps
-            with self.logger.accumulate_means("agent"):
-                self.logger.log(f"Training agent for {num_steps} timesteps")
-                self.trajectory_generator.train(steps=num_steps)
+            self.logger.log(f"Training agent for {num_steps} timesteps")
```

Comment:
If I understand correctly, entering `accumulate_means` is moved off to the trajectory generator, and calls to `log` do not use the prefix/name system as it's not a recorded quantity. Is this correct?

Reply:
That is correct.